site/content/en/docs/Guides/Client SDKs/rest.md

```diff
@@ -238,6 +238,97 @@ Apply an Annotation with the prefix "agones.dev/sdk-" to the backing `GameServer
 curl -d '{"key": "foo", "value": "bar"}' -H "Content-Type: application/json" -X PUT http://localhost:${AGONES_SDK_HTTP_PORT}/metadata/annotation
 ```
 
+### Counters
+
+{{< alpha title="Counters" gate="Counters" >}}
+
+#### Alpha: GetCounter
+This function retrieves a specified counter by its name and returns its information.
+
+##### Example
+
+```bash
+curl -H "Content-Type: application/json" -X GET http://localhost:${AGONES_SDK_HTTP_PORT}/v1alpha1/counters/conformanceTestCounter
```

Comment:
We should probably mention in the example that conformanceTestCounter is predefined in the local SDK Server, or go with the more generic example of {name} or {counterName}.

Reply:
I have included the Local Development Setup section for Counters and Lists in this document. Could you please review it and share your thoughts?

Reply:
I would add an explainer under each example that uses a key, something like:

> In this example, we retrieve the counter under the key "conformanceTestCounter" to make it extra clear.

Since people don't always read the top level docs 😃

As an improvement to the docs and likely the local sdk experience,  @igooch I didn't catch this earlier, but any objection to changing:
https://github.com/googleforgames/agones/blob/1e78d88a27ae7d427e43f073aab7d28289b7d850/pkg/sdkserver/localsdk.go#L147-L158

To using some more generic keys? Maybe `players` for the list and `rooms` for the count? 

We don't actually use these values for conformance tests anyway, so this might be a bit confusing, and I think would make examples easier to understand. WDYT? If so, we can do this in a separate PR (and will need to feature shortcode the new docs) - if you like we can file a new issue.

Reply:
>I would add an explainer under each example that uses a key, something like:
In this example, we retrieve the counter under the key "conformanceTestCounter" to make it extra clear.

I have included this information in a `Note` section located under both the `Counters` and `Lists` titles, following the `Warning` section. This will avoid the repetition right? WDYT?

>To using some more generic keys? Maybe players for the list and rooms for the count?

Do you want to replace `"conformanceTestCounter": {Count: 1, Capacity: 10}} ` with `"rooms": {Count: 1, Capacity: 10}} ` and `"conformanceTestList": {Values: []string{"test0", "test1", "test2"}, Capacity: 100}}` with `"players": {Values: []string{"player1", "player2", "player3"}, Capacity: 100}}`?

Reply:
> We don't actually use these values for conformance tests anyway

I'm silly - of course we use this for conformance tests 🤦🏻 . Ignore that comment. But I am wondering if we change the key anyway, to make it easier for end users, and make our conformance testing match. WDYT?

For these docs, we don't have to conform to the local SDK for the example, so I think we can change the key to something more generic like I suggest above. I'd like to get agreement on the code changes from @igooch before we make the code changes though to the local sdk server (and either way, we should handle that in a separate PR).

Sound good?

Reply:
My personal preference is to have the example match what we have in the localSDKServer for a better user experience. That way the user can run the example code and it works without them needing to make changes. My general gripe with generic examples is that they implicitly assume the user knows what and where to make changes without any accompanying instructions. That being said the key "conformanceTestCounter" probably doesn't mean anything to the user either, so I'd rename if along the lines of "TestCounterKey", "TestPlayerCounter", etc.

I'm also fine leaving the examples more generic here, as long as we have a step by step guide elsewhere that is more accessible to the user that needs more context on how and where to make changes.

Reply:
> My personal preference is to have the example match what we have in the localSDKServer for a better user experience.

100% agreed - no disagreement there - sounds like we're just talking about what names to give keys. This change I've suggested above I would like to see as an interim step until we can align it again next release with a future PR to update the local sdk server and conformance tests.

> That being said the key "conformanceTestCounter" probably doesn't mean anything to the user either, so I'd rename the keys along the lines of "TestCounterKey", "TestPlayerCounter", etc.

So i would argue that both "test", "key", and "counter" are redundant in the name, and just confuse things a bit more, and it's easier to give a real world use case for the local SDK user that is easy to understand (and will align with the eventual integration pattern docs I'll write - primarily one for "rooms" and one for "player tracking" - as well as the examples we have in the yaml files etc.).

That being said, I would be super happy if as part of the description for each method, there is a reminder for what the key is that is being used, since reminders are great and redundancy in docs is definitely not a bad thing, I'll write up a small suggestion, lemme know what you think.

Reply:
Are we sticking with `rooms` and `players` for now and considering changes later if needed?